playwright/components/radio-button/locators.ts

```diff
@@ -0,0 +1,5 @@
+// component preview locators
+export const RADIOBUTTONGROUP = '[data-component="radiogroup"]';
```

Comment:
**suggestion:** I know you've copied these across as they are but I'm pretty sure in other places we have an underscore between the words. It would be `RADIO_BUTTON_GROUP`. To me it's a lot easier to read too.